app/enterprise/1.3-x/deployment/installation/amazon-linux-2.md

```diff
@@ -1,343 +1,320 @@
---- 
-title: Install Kong Enterprise with Amazon Linux 2
+---
+title: How to Install Kong Enterprise on Amazon Linux 2
```

Comment:
Suggest we remove "How to" from our topics, as how to is implied with the "install" action. Change to "Install Kong on Amazon Linux 2"

Reply:
I agree, it's implied. It's possible @felderi and the co were following our Technical Writing Guide, which needs to be updated now.

Reply:
https://github.com/Kong/docs.konghq.com/blob/master/TECHNICAL-WRITING-GUIDE.md#how-to-guides

Reply:
Done.